pandas/core/arrays/boolean.py

```diff
@@ -101,6 +101,24 @@ def __repr__(self) -> str:
     def _is_boolean(self) -> bool:
         return True
 
+    def __from_arrow__(self, array):
```

Comment:
can this be annotated?  or at least have types in the docstring

Reply:
The type is mentioned in the docstring (just not in a parameters section). No one except for pyarrow should be calling this method though.

Question for annotating: how does it work to annotate it with pyarrow objects that cannot necessarily be imported? (since it's an optional dependency)

Reply:
Any idea about the annotation?